tests/faultydata/variants/duplicated_sampleid/README.md

```diff
@@ -0,0 +1,6 @@
+Data file example_1.vcf.gz was copied from tests/data/variants/example_1, then change sample id S2 to S1
+
+Index file was generated by
+```
```

Comment:
I don't have a strong opinion here. You don't need to understand markdown to read it as plain text. We're already using restructured text, markdown and plain text for the READMEs in various places, so we're not doing anything new here either way. It's up to you guys!